### ISSUE TITLE
ruby var and def...end same has same color

### ISSUE THREAD

**Issue by yueqianzhang** [srcery-colors/srcery-vim#29]
Is it correct ?

**Comment by yueqianzhang** [comment]
![image](https://user-images.githubusercontent.com/20694300/47572925-b7517900-d96e-11e8-8668-c4f85b5f3154.png)

**Comment by roosta** [comment]
Yes this is correct, there isn't much ruby specific highlighting in Srcery beyond what is defined in general syntax highlighting. From what I could tell ruby.vim links `rubyDefine` and `rubyClassVariable`  to `Define` and `Identifier` respectively, which are both cyan in Srcery. I did play around with changing `rubyDefine` to red, but I don't code ruby so I don't really know if it works. What do you think?
![rubyred](https://user-images.githubusercontent.com/4509009/47578371-8579eb80-d949-11e8-82bd-f570b492e0d0.png)
